infra/gcp/clusters/projects/k8s-infra-prow-build-trusted/prow-build-trusted/resources/ghproxy-ingress.yaml

```diff
@@ -0,0 +1,11 @@
+apiVersion: networking.k8s.io/v1beta1
+kind: Ingress
+metadata:
+  name: ghproxy
+  namespace: default
+  annotations:
+    kubernetes.io/ingress.global-static-ip-name: "ghproxy-ingress-prod"
+spec:
+  backend:
+    serviceName: ghproxy
+    servicePort: main
```

Comment:
We do not want this exposed as a service outside of the cluster at all; we should only be caching for services and jobs deployed in this cluster, not the internet.

Please take a look at how ghproxy is currently deployed for prow: https://github.com/kubernetes/test-infra/blob/master/config/prow/cluster/ghproxy.yaml